### ISSUE TITLE
Return to previous camera view on stream disconnect

### ISSUE THREAD

**Issue by r-barton** [KittyCAD/modeling-app#4254]
Whenever the stream disconnects, the camera view of the modeling app does not return to the previous camera position. Instead it resets the view as if the user had selected the `Reset view` button in the UI. This can be quite jarring as the end user

**Comment by franknoirot** [comment]
@nadr0 will #5532 help with this by any chance? This is probably simpler than that, just saving and applying the last camera position in case of a reconnect.

**Comment by lf94** [comment]
stream idle is already on this heh.

**Comment by nadr0** [comment]
Correct, we will be able to have a last saved view and restore it correctly with the engine APIs that named views use.
We don't need a `Named View` you can use the API's that it uses to save that off in memory and restore it later.
